docs/en-US/component/cascader.md

```diff
@@ -109,9 +109,9 @@ cascader/filterable
 
 ## Custom option content
 
-You can customize the content of cascader node.
+You can customize the content of cascader node and suggestion items.
 
-:::demo You can customize the content of cascader node by `scoped slot`. You'll have access to `node` and `data` in the scope, standing for the Node object and node data of the current node respectively.
+:::demo You can customize the content of cascader node by `default` slot. You'll have access to `node` and `data` in the scope, standing for the Node object and node data of the current node respectively. Additionally, you can use the `suggestion` slot to customize the content of suggestion items when filtering, with access to the suggestion `item` in the scope.
```

Comment:
We can add a new paragraph and example here to describe the new slot without affecting the original description and example.